### ISSUE TITLE
Add/recreate documentation from the original project

### ISSUE THREAD

**Issue by dandv** [brix/crypto-js#97]
The docs at https://code.google.com/archive/p/crypto-js/ are abandoned and look crappy on that site (no syntax highlight, terrible formatting).

I'd volunteer to bring them over, but first I'd like to know the project is [still maintained](https://github.com/brix/crypto-js/pull/96).

**Comment by evanvosberg** [comment]
Yes the project is still maintained, but not heavy under development. It's more about small fixes ...
